### PR TITLE
[FEATURE] Document how api.typo3.org can be updated

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Maintainers/Tools.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.. include:: /Includes.rst.txt
+.. highlight:: shell
+.. index:: Changes; Backporting
+.. _backport-changes:
+
+=====================
+Tools of the Doc Team
```
[the remark this answers is not in the document]
Comment (linawolf) [reply Documentation/Maintainers/Tools.rst:7]: Documentation is similar so it should be Doc Team?

File: Documentation/Maintainers/Tools.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Landing page
+------------
+
+The landing page of https://api.typo3.org/ is automatically updated when this
+file is updated in GitHub:
+
+https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/blob/main/WebRootResources-api.typo3.org/index.html
+
+As this is a pure HTML file, we currently keep the content in this page and
```
Comment (linawolf) [inline Documentation/Maintainers/Tools.rst:21]:
```suggestion
As this is a pure HTML file, we currently keep the content in this rst file:
https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/blob/main/Documentation/Home/ApiTypo3Org.rst
localy render it, 
```

File: Documentation/Maintainers/Tools.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/Maintainers/Tools.rst:7]: Hmm, I think we should write it as full "Documentation Team". We are no doctors ;-) And we have the place.

File: Documentation/Maintainers/Tools.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
+.. include:: /Includes.rst.txt
+.. highlight:: shell
+.. index:: Changes; Backporting
+.. _backport-changes:
+
+=====================
+Tools of the Doc Team
+=====================
```
Comment (linawolf) [inline Documentation/Maintainers/Tools.rst:8]:
```suggestion
===============================
Tools of the Documentation Team
===============================
```

File: Documentation/Maintainers/Tools.rst [hunk 4]
```diff
@@ -18,7 +18,9 @@ file is updated in GitHub:
 
 https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/blob/main/WebRootResources-api.typo3.org/index.html
 
-As this is a pure HTML file, we currently keep the content in this page and
+As this is a pure HTML file, we currently keep the content in this rst file:
+https://github.com/TYPO3-Documentation/DocsTypo3Org-Homepage/blob/main/Documentation/Home/ApiTypo3Org.rst
+localy render it,
```
Comment (brotkrueml) [inline Documentation/Maintainers/Tools.rst:23]:
```suggestion
locally render it, 
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**